wagtail/contrib/search_promotions/views.py

```diff
@@ -19,16 +20,30 @@
 )
 @vary_on_headers('X-Requested-With')
 def index(request):
-    is_searching = False
-    query_string = request.GET.get('q', "")
+    # Ordering
+    valid_ordering = ['query_string', '-query_string', 'ohits', '-ohits']
```

Comment:
At first read the meaning of the query string parameter "ohits" isn't really clear. Since under the hood this is an annotated value that doesn't have to line up to an existing model field, it might be nicer if the query string parameter matched what is displayed in the column header, maybe "views".

Reply:
@chosak  - thanks for the feedback, I agree that ohits was a bit confusing. I have renamed to `views`